bin/auto-reveal.js

```diff
@@ -47,8 +47,31 @@ async function start() {
 	server.bindCLIShortcuts({ print: true });
 }
 
-if (!argv[0] || argv[0] === 'start') {
-	start();
-} else if (argv[0] === 'build') {
-	build(config);
-}
+const program = new Command();
+
+program.name(process.env.npm_package_name);
+program.description(process.env.npm_package_description);
+program.version(process.env.npm_package_version);
+
+program
+	.command('start', {
+		isDefault: true,
+	})
+	.description('Live-reloading server for your slides.')
+	.action(start);
+
+program
+	.command('build')
+	.description('Build a static copy of your presentation.')
+	.option('-o, --outDir <path>', 'Output directory', config.build.outDir)
+	.action((options) =>
+		build({
+			...config,
+			build: {
+				...config.build,
+				outDir: path.resolve(cwd, options.outDir),
```

Comment:
I think this is going to give you a different answer than you expect 🤔 since there is no default set for the outDir I'm assuming it's going to be undefined and `path.resolve(cwd, undefined)` I think just gives you cwd no? maybe we just define the default ourselves instead of relying on vite's default?

Reply:
oh I misread this, ignore me! I didn't realise that we had already produced the outdir in the code above. It begs the question why we need to do a resolve here tho 🤔

Reply:
There is a default set, but I made that a bit more verbose.